lib/screens/asignaturas_lista_screen.dart

```diff
@@ -132,3 +124,65 @@ class _AsignaturasScreenState extends State<AsignaturasScreen> {
     );
   }
 }
+
+class AsignaturaListTile extends StatelessWidget {
+  const AsignaturaListTile({
+    Key? key,
+    required this.asignatura,
+  }) : super(key: key);
+
+  final Asignatura asignatura;
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = MainTheme.theme;
+
+    return InkWell(
```

Comment:
Hay un detalle con este `InkWell`, el efecto ripple (que es cuando mantienes presionado) queda por fuera de la cajita, y debería estar en la cajita (con bordes redondeados y todo). Puedes mirar las `Card` del home para cachar como debería quedar, y puedes echarle un ojo a [esta pregunta de StackOverflow](https://stackoverflow.com/questions/45424621/inkwell-not-showing-ripple-effect) que yo siempre miro para hacerle el efecto jeje

Reply:
No puedo sacarle pantallazo, pero logré el ripple effect dentro de la tarjeta en el commit [50048ea](https://github.com/exdevutem/mi-utem/pull/29/commits/50048eaa2ac86d3ad74129efffb1674eb6538311) solamente intercambiando el inkwell dos widgets para abajo en la jerarquía